Add description for install issue for using swap partition on drive (#39842)
home-assistant/home-assistant.github.io: source/_includes/installation/operating_system.md

      - If you are getting an **Error unmounting filesystem** error message, stating that the **target is busy**:
      - Most likely, you are running Ubuntu on your internal disk. Instead, you need to run it on your stick.
        - Go back to step 3 and during start up, make sure you select **Try Ubuntu** (and NOT **Install Ubuntu**).

>       - Another issue may be that live Ubuntu are using Swap partition from existing Linux install.

Reviewer: ```suggestion
      - Another issue may be that live Ubuntu is using the Swap partition of an existing Linux installation.
```

>         If you see "Swap" listed as a partition on the drive you're going to wipe, just select it, then press the stop button to unmount it and try again.

Reviewer: ```suggestion
        - If you see "Swap" listed as a partition on the drive you're going to install HAOS, just select the Swap partition, then press the stop button to unmount it and try the restore operation again.
```
